google-cloud-pubsub/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+### 0.37.0 / 2019-06-17
+
+* Add Topic#persistence_regions
+* Subscriber changes
+  * Fix potential inventory bug
+    * It is possible 
```

Comment:
"It is possible"?

Is this entry for #3450?

Reply:
Fixed.